lib/internal/modules/esm/initialize_import_meta.js

```diff
@@ -7,14 +7,14 @@ const experimentalImportMetaResolve = getOptionValue('--experimental-import-meta
  * Generate a function to be used as import.meta.resolve for a particular module.
  * @param {string} defaultParentUrl The default base to use for resolution
  * @param {typeof import('./loader.js').ModuleLoader} loader Reference to the current module loader
- * @returns {(specifier: string, parentUrl?: string) => string} Function to assign to import.meta.resolve
+ * @returns {(specifier: string, parentURL?: string) => string} Function to assign to import.meta.resolve
  */
 function createImportMetaResolve(defaultParentUrl, loader) {
-  return function resolve(specifier, parentUrl = defaultParentUrl) {
+  return function resolve(specifier, parentURL = defaultParentUrl) {
     let url;
 
     try {
-      ({ url } = loader.resolveSync(specifier, parentUrl));
+      ({ url } = loader.resolveSync(specifier, parentURL));
```

Comment:
Let's revert that and take it to its own PR since it's not related
```suggestion
      ({ url } = loader.resolveSync(specifier, parentUrl));
```